infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/StreamsSnapshotWriter.java

```diff
@@ -242,6 +242,15 @@ public void apply(LogReplicationEntryMsg message) {
         }
         UUID regularStreamId = opaqueEntry.getEntries().keySet().stream().findFirst().get();
 
+
+        // When there is an upgrading and Active's streams to replicate set has streams not present
+        // in Standby's set, they will be dropped for version compatibility consideration
+        if (!regularToShadowStreamId.containsKey(regularStreamId)) {
+            log.info("Stream {} sent from Active is not expected in Standby. LR could be in upgrading", regularStreamId);
```

Comment:
can you reword it to - ".. not expected on Standby.  LR could be undergoing a rolling upgrade."

Reply:
Changed, thanks!